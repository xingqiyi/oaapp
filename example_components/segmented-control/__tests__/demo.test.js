import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('segmented-control');
