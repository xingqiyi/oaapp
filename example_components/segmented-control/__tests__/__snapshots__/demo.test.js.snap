// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/segmented-control/demo/basic.md correctly 1`] = `
<div
  class="am-wingblank am-wingblank-lg sc-example"
>
  <p
    class="sub-title"
  >
    Simplest
  </p>
  <div
    class="am-segment"
    role="tablist"
  >
    <div
      aria-disabled="false"
      aria-selected="true"
      class="am-segment-item am-segment-item-selected"
      role="tab"
      style="color:#fff;background-color:;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换一
    </div>
    <div
      aria-disabled="false"
      aria-selected="false"
      class="am-segment-item"
      role="tab"
      style="color:;background-color:transparent;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换二
    </div>
  </div>
  <p
    class="sub-title"
  >
    Disabled
  </p>
  <div
    class="am-segment am-segment-disabled"
    role="tablist"
  >
    <div
      aria-disabled="true"
      aria-selected="false"
      class="am-segment-item am-segment-item-selected"
      role="tab"
      style="color:#fff;background-color:;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换一
    </div>
    <div
      aria-disabled="true"
      aria-selected="false"
      class="am-segment-item"
      role="tab"
      style="color:;background-color:transparent;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换二
    </div>
  </div>
  <p
    class="sub-title"
  >
    SelectedIndex
  </p>
  <div
    class="am-segment"
    role="tablist"
  >
    <div
      aria-disabled="false"
      aria-selected="false"
      class="am-segment-item"
      role="tab"
      style="color:;background-color:transparent;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换一
    </div>
    <div
      aria-disabled="false"
      aria-selected="true"
      class="am-segment-item am-segment-item-selected"
      role="tab"
      style="color:#fff;background-color:;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换二
    </div>
    <div
      aria-disabled="false"
      aria-selected="false"
      class="am-segment-item"
      role="tab"
      style="color:;background-color:transparent;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换三
    </div>
  </div>
  <p
    class="sub-title"
  >
    TintColor
  </p>
  <div
    class="am-segment"
    role="tablist"
    style="height:0.8rem;width:5rem;"
  >
    <div
      aria-disabled="false"
      aria-selected="true"
      class="am-segment-item am-segment-item-selected"
      role="tab"
      style="color:#fff;background-color:#ff0000;border-color:#ff0000;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换一
    </div>
    <div
      aria-disabled="false"
      aria-selected="false"
      class="am-segment-item"
      role="tab"
      style="color:#ff0000;background-color:transparent;border-color:#ff0000;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换二
    </div>
    <div
      aria-disabled="false"
      aria-selected="false"
      class="am-segment-item"
      role="tab"
      style="color:#ff0000;background-color:transparent;border-color:#ff0000;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换三
    </div>
  </div>
  <p
    class="sub-title"
  >
    onChange/onValueChange
  </p>
  <div
    class="am-segment"
    role="tablist"
  >
    <div
      aria-disabled="false"
      aria-selected="true"
      class="am-segment-item am-segment-item-selected"
      role="tab"
      style="color:#fff;background-color:;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换一
    </div>
    <div
      aria-disabled="false"
      aria-selected="false"
      class="am-segment-item"
      role="tab"
      style="color:;background-color:transparent;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换二
    </div>
    <div
      aria-disabled="false"
      aria-selected="false"
      class="am-segment-item"
      role="tab"
      style="color:;background-color:transparent;border-color:;"
    >
      <div
        class="am-segment-item-inner"
      />
      切换三
    </div>
  </div>
</div>
`;
